public_html/assets/js/nf-core-schema-builder.js

```diff
@@ -1672,16 +1672,16 @@ function clean_empty_schema_keys(subschema){
 function clean_empty_param_keys(param){
 
     // Clean up empty strings
-    if(param.hasOwnProperty('description') && param['description'] == ''){
+    if(param.hasOwnProperty('description') && param['description'] === ''){
         delete param['description'];
     }
-    if(param.hasOwnProperty('default') && param['default'] == ''){
+    if(param.hasOwnProperty('default') && param['default'] === ''){
```

Comment:
This is the only place I am not 100% sure, because this changes some behavior to before, where everything evaluating to `false`, e.g. `0`, but also `false` was removed, @ewels

Reply:
Hmm, yes this might need a little more code then. I think that we want to remove empty strings and boolean `false` but maybe not numeric `0`?

```suggestion
    if(param.hasOwnProperty('default') && (param['default'] === '' || param['default'] === false)){
```

Untested - what do you think?